dependency-manager-tests/spm/SPMProject/MasterViewController.swift

```diff
@@ -0,0 +1,194 @@
+//
+//  MasterViewController.swift
+//  SPMProject
+//
+//  Created by Mert Buran on 26/03/2020.
+//  Copyright © 2020 Datadog. All rights reserved.
+//
+
+import UIKit
+import CoreData
```

Comment:
CoreData 💔. How about using "Single View App" with just `UITests` enabled and no Core Data nor unit tests, as we won't need them?

Reply:
I feel you 😄 
I kept it checked at project creation in order to make the project more realistic, it doesn't seem to cause any problems so far.
Adding unit tests afterwards usually cause me pain, that's why I kept it in although as you said we don't use them for now.

To be honest, I didn't pay attention to details as we are planning to change the project structure soon. 
Shall I make these changes now or leave for later?

Reply:
Sure, we can tackle it later when doing `UITests` for SPM project 👌🙂.